src/main/java/me/drepic/proton/Proton.java

```diff
@@ -36,8 +36,16 @@ public void onEnable(){
         saveConfig();
 
         String clientName = config.getString("identification.clientName");
+        assert clientName != null;
```

Comment:
Asserts are disabled by default in Java. You can manually throw an exception or use something like Guava Preconditions.